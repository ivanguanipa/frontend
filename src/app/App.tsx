import * as React from 'react';
import { appRouter as AppRouter } from 'app/AppRouter';
import { GlobalErrorBoundary } from './core/errors/GlobalErrorBoundary';
import { Provider } from 'react-redux';
import store from 'app/core/redux/store';

function app() {
  return (
    <GlobalErrorBoundary>
      <Provider store={store}>
        <AppRouter />
      </Provider>
    </GlobalErrorBoundary>
  );
}

export default app;
