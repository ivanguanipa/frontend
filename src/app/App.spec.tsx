import * as React from 'react';
import { ShallowWrapper, shallow } from 'enzyme';
import App from './App';

describe('App Page Test', () => {
    let componentWrapper: ShallowWrapper;
    afterEach(() => {
            componentWrapper.unmount();
        });

    it('should match snapshot', () => {

        componentWrapper = shallow(
          <App/>
        );
        expect(componentWrapper).toMatchSnapshot();
      });
})