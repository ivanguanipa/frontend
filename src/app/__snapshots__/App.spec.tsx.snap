// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Page Test should match snapshot 1`] = `
<GlobalErrorBoundary>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <appRouter />
  </Provider>
</GlobalErrorBoundary>
`;
